### DISCUSSION TITLE
Date time in non ISO format are not flagged in examples

### DISCUSSION THREAD

**Discussion by savage-alex** [stoplightio/spectral#2431] [Rulesets]
Given the following query parameter

![image](https://user-images.githubusercontent.com/25430683/221230883-b434b4bf-66e8-4b85-bca3-a8d2a90331d4.png)

I would like to be informed that the example isnt valid date time like it does when a date is missing
![image](https://user-images.githubusercontent.com/25430683/221231042-f938bd54-1862-48b3-90ad-633b99fbb668.png)
so that my examples are valid date times and not missing the T and time zone component

**Comment by emi-emi671** [comment]
Did you find any ruleset which validates the date-time format?

**Comment by savage-alex** [comment]
the core OAS3 validates the date-time which is shown in screenshot 2. I think it perhaps could be more opinionated on what valid is like

**Comment by emi-emi671** [comment]
OK, Currently, my ruleset `extends: "spectral:oas"` So you mean to verify date-time in ISO8601 I should need to `extends: spectral:oas3` ?

**Comment by savage-alex** [comment]
sorry my text wasnt clear. it is indeed "extends: spectral:oas"
